.github/workflows/coding-standards.yml

```diff
@@ -13,6 +13,11 @@ on:
         description: "The version the package being tested, in case of circular dependencies."
         required: false
         type: "string"
+      composer-options:
+        description: "Additional flags for the composer install command."
+        default: "--prefer-dist"
```

Comment:
This is the default behavior according to `composer help update`. Any reason not to use `""`?

Reply:
It overrides potential settings in composer.json: https://getcomposer.org/doc/06-config.md#preferred-install

Reply:
In the unlikely event that we need to resort to it, why would we want to prevent ourselves from using this?

Reply:
I think composer only accepts these settings in the root composer.json. So if someone would use this configuration in a library, running composer install as part of that library's unit tests would work, but installation could fail for end uses of the library. My understanding of `--prefer-dist` is that this ensures that such a scenario cannot happen.

Reply:
> So if someone would use this configuration in a library

These workflows can only be used within the doctrine organization for now.
I can't think why maintainers would resort to that kind of settings, and since they currently don't, we don't have a way to prove this works as expected. IMO, YAGNI.

Reply:
@driehle Oh sorry, I didn't notice `--prefer-dist` was already what we were using as a default! I was wondering why you insisted on adding it :sweat_smile: 
Let's use it indeed, otherwise it will be a BC-break!

Reply:
Yes, there has already been a `--prefer-dist`, it just moved to `composer-options` now, so that people have a chance of removing it, if they like.